src/main/java/ganymedes01/etfuturum/entities/EntityNewBoatSeat.java

```diff
@@ -115,10 +115,6 @@ public void onUpdate() {
 		copyLocationAndAnglesFrom(boat);
 	}
 
-	private void mountToBoat(Entity entity) {
-		boat.sitEntity(entity);
-	}
-
```

Comment:
Unused ≠ Dead
Please refrain from removing utility code that may be used in the future.